internal/proposals/logs/dynamic-backend-configuration/custom-operator/dynamic-fluentbit-config.md

```diff
@@ -0,0 +1,148 @@
+
+An alternative for `kubebuilder` would be the [`Operator-SDK`](https://github.com/operator-framework/operator-sdk) from `Red Hat`. This has similar features to the kubebuilder, with some additions like the integration of [`operatorhub`](https://operatorhub.io/) - but such features aren't needed for our purpose. Furthermore, this isn't an upstream project of Kubernetes.
+
+To have a simple API, one `CRD` for Fluent Bit configuration is created. This includes a field to determine the type of the configuration (`FILTER` or `OUTPUT`). A list of maps for the Fluent Bit configuration code, and a status field to keep track of the status of the later created CR.
+Using a list of maps for the actual configuration, enables the possibility to support the full Fluent Bit syntax without having the need to maintain new features of various plugins. Furthermore, this gives the users the ability to have a good overview of their sequence of applied filters and outputs. Using the Kyma documentation, we could also lead the users to think more in a way of pipelines, in such that they create one CR for each Fluent Bit pipeline.
+
+One constraint of this operator is, that it doesn't support dynamic plugins, which require to be loaded into the Fluent Bit image. Another constraint is, that not all available plugins for filters and outputs can be configured by this operator (i.e. `Lua`), because such plugins need to have certain files mounted into the container, which isn't possible. If users still want to use such, they need to configure an output plugin of their choice and process the logs with another log processor (i.e. another instance of Fluent Bit). The operator has a list of unsupported plugins, to achieve that such plugins can't be configured. Thus, every time users create a CR, the list is checked if it contains a wanted plugin, if it does users are informed that they aren't able to create such CR.
+
+<details>
+<summary><b>Pipeline Overview</b> for User - Click to expand</summary>
+
+![Thank you](images/fluentbit_CR_overview.svg)
+</details>  
+
+The logs are fetched using a Fluent Bit  , which is created by this operator. This INPUT tags all logs with the following tag scheme: `kube.<namespace_name>.<pod_name>.<container_name>`. Afterwards, the logs are enriched with Kubernetes information using the `kubernetes` filter of Fluent Bit. Then, the filter `rewrite_tag` is used to split the pipeline. The original logs are forwarded to the Kyma Loki backend, and the new copy with another tag can be used by the users by configuring new filters and outputs with the provided CRD. Using this approach, we avoid the situation of an unused INPUT or other overhead.
```

Comment:
```suggestion
1. The logs are fetched using a Fluent Bit, which is created by `kubebuilder`. This INPUT tags all logs with the following tag scheme: `kube.<namespace_name>.<pod_name>.<container_name>`. 
2. The logs are enriched with Kubernetes information using the `kubernetes` filter of Fluent Bit. 
3. The filter `rewrite_tag` is used to split the pipeline:
   - The original logs are forwarded to the Kyma Loki backend.
   - Users can use the new copy with another tag and configure new filters and outputs with the provided CRD. 

Using this approach, we avoid having an unused INPUT or other overhead.
```

Reply:
This custom operator is not called `kubebuilder`. The SDk used for it is called like this. I will change the suggestion abit and will commit